lib/src/base-styles/_nx-toggle.scss

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../scss-shared/nx-variables';
+@import '../scss-shared/nx-text-helpers';
+@import '../scss-shared/nx-container-helpers';
+
+.nx-toggle {
+  @include container-horizontal;
+  @include regular();
+
+  display: block;
+  margin-bottom: $nx-spacing-xs;
+  white-space: nowrap;
+  max-width: max-content;
+
+  &:hover {
+    cursor: pointer;
+
+    .nx-toggle__control {
+      border-color: #2c2c2c;
+      filter: none;
+      stroke: #2c2c2c;
+    }
+  }
+
+  &:focus-within {
+    .nx-toggle__control {
+      border-color: #02a6d9;
+      stroke: #02A6D9;
```

Comment:
I think we use lowercase colors everywhere

Reply:
> I think we use lowercase colors everywhere

https://github.com/sonatype/sonatype-react-shared-components/pull/117/commits/502a52f1a486eadfa699634663a60861f10b0d28